### ISSUE TITLE
Vertical scroll on product screenshot makes the page scroll down

### ISSUE THREAD

**Issue by adnan449** [elementary/appcenter#1589]
## What Happened

While scrolling down on app screenshots, appcenter scrolls down and swipes right on the screenshot gallery. 

## Expected Behavior

Not scroll down the page and only swipe the screenshots.

## Steps to Reproduce

1. Browse an app in the appcenter
2. On the screenshots, scroll down using your mouse wheel
3. See the What Happened section

## Platform Information

* The version of elementary OS you are using, or other operating system

Elementary OS 6

* The version of the software you are using such as "1.0", "Compiled from git", or "Latest release" if you're not sure but you have run updates

3.6.2

[Video](https://streamable.com/p7f6dd)

**Comment by cassidyjames** [comment]
Rather than hijack scrolling, the solution I'd like to see is to disable vertical mouse wheel scrolling on the horizontal screenshots. We have pager buttons, horizontal scroll support, and swipe support, but scrolling down the page shouldn't be interrupted; this is how this is typically implemented on the web as well, for example.

We're blocked by the release of Handy 1.3, which should be coming along with GNOME 41 in ~a month.

**Comment by cassidyjames** [comment]
Oops, duplicate of #1578
